### PR TITLE
GSoC 2022 Student Blog for the ATHENA Project

### PR DESCRIPTION
**Pull request by Ajax-Light**
My Student Blog for GSoC 2022 - Mid-Term Evaluations

### CODE REVIEW

**Comment by Ajax-Light** [comment]
@wdconinc Please Review

File: _gsocblogs/2022/blog_ATHENA_UjwalKundur.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+The project is pretty straight-forward, at least on the surface; Porting a clustering algorithm used in the reconstruction of particle collision events at the Electron-Ion Collider (**EIC**) to **SYCL**.
+The clustering algorithm needs to scale in order to process increasing number of hits as the Sensor resolution is upgraded over the years, this requires hardware acceleration which is already being used significantly in AI/ML to train models using GPUs. Other hardware accelerators do exist, for example, FPGAs and ASICs (Application Specific Integrated Circuits) which are prominently used in Bitcoin-mining. We stick to GPUs in this project.
+
+SYCL aims to make programmers' lives easier by allowing us to target all these different hardware architectures from a single codebase. While many implementations of the SYCL standard exist, similar to how there are many implementations of *ISO C*, Intel's offering - **Intel OneAPI DPC++** (Data Parallel C++) seems to be the most mature project and we stick to it for our SYCL needs.
```
Comment (hegner) [inline _gsocblogs/2022/blog_ATHENA_UjwalKundur.md:22]: lifes

**Review by hegner — CHANGES_REQUESTED** [review]
Just fix that typo and we will add it to the blog

File: _gsocblogs/2022/blog_ATHENA_UjwalKundur.md (hunk 1 above)
Comment (Ajax-Light) [reply _gsocblogs/2022/blog_ATHENA_UjwalKundur.md:22]: Hi @hegner I don't think that's a typo ( I did a proof-read before the PR ). https://www.queens-english-society.com/lives-vs-lifes
image

Comment (Ajax-Light) [reply _gsocblogs/2022/blog_ATHENA_UjwalKundur.md:22]: However, Lemme know if I have to correct it

Comment (hegner) [reply _gsocblogs/2022/blog_ATHENA_UjwalKundur.md:22]: You are correct!

**Pull request merged**